weather-forecast/ecrad-3d-correction/unets/configs/unet.yaml

```diff
@@ -30,6 +30,6 @@ data:
 
 trainer:
     max_epochs: 50
-    accelerator: "cpu"
+    accelerator: "gpu"
     devices:
         - 0
```

Comment:
Why sometimes devices is 1 and sometimes 0 ?

Reply:
Because it is not the same meaning. Using cpu must be un int > 0 because you chose the number of accelerator but not where there are placed. Using a gpu ones, you can chose the index of the accelerator (that start at 0).